attributes/default.rb

```diff
@@ -22,13 +22,12 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-# AWS Placeholders
-default['incident_bot']['aws'].tap do |aws|
-  aws['redis_bucket'] = ''
-  aws['secret_key'] = ''
-  aws['access_key'] = ''
-  aws['domain'] = ''
-end
+# AWS
+default['incident_bot']['aws']['domain'] = ''
```

Comment:
Ummm, it works for now :-) so... maybe next feature release? We can create an issue to track, but I'm going to be lazy for now.